IntGeVaP/README.md

```diff
@@ -42,12 +42,12 @@ autopvs1 file (input/*autopvs1.txt)
 
```

Comment:
We need to add in variant_summary.txt.gz AND submission_summary.txt.gz 
Also need to mention, those two files need to be unzipped.